### ISSUE TITLE
508-defect-1 ❗ Launchblocker [SCREENREADER]: In footer, Tagalog and Espanol are pronounced in an english accent 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25593]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

<del>There appears to be a bug where the string "href" is appended to the `lang` attribute. This causes the screen reader to announce "Espanol" and "Tagalog" in English. Removing the "href" ahead of "lang" will fix the issue. </del>

Edit: not a bug, just missing the `lang` attribute which is separate to the `hreflang`

> Hints at the human language of the linked URL. No built-in functionality. Allowed values are the same as the global lang attribute.

## Acceptance Criteria

- [ ] `lang` attribute is added on top of the `hreflang`

## Screenshots or Trace Logs

<img width="1516" alt="Screen Shot 2021-06-03 at 9 22 49 AM" src="https://user-images.githubusercontent.com/14154792/120652298-a5e97b00-c44d-11eb-838a-4936c4d527fa.png">

**Comment by joshkimux** [comment]
<img width="1516" alt="Screen Shot 2021-06-08 at 10 49 43 AM" src="https://user-images.githubusercontent.com/14154792/121207394-3dccd780-c847-11eb-9281-918e0138588b.png">

It looks like depending on the language of the full page, the links in the footer are being affected as well.

<img width="1504" alt="Screen Shot 2021-06-08 at 10 50 38 AM" src="https://user-images.githubusercontent.com/14154792/121207538-5d640000-c847-11eb-9a58-22742ec8fa87.png">

English should not be pronounced in a Spanish or Tagalog accent.

**Comment by joshkimux** [comment]
This has been fixed! Great work 🥳
